episodes/17-operations.md

```diff
@@ -35,22 +20,143 @@ exercises: 5
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+Until now, this training has focused on the internals of a lesson development project.
+Now it is time to consider the context in which this lesson development can take place.
+
+## The Carpentries Lesson Development Ecosystem
+
+[The Carpentries Incubator][carpentries-incubator] is a space for The Carpentries community
+to collaborate on lessons.
+It provides a single location where our community can come to 
+find lessons that are being developed by the community,
+and to create new lessons using The Carpentries Workbench.
+Community members retain ownership of their lessons hosted in the Incubator,
+while benefiting from the increased findability it gives to the lesson.
+
+The Carpentries supports the lesson developer community working in the Incubator
+with training (like this one),
+communications designed to raise awareness of lesson projects within the community,
+and platforms for the community to discuss and collaborate on lesson development.
+
+The Carpentries also provides a space for community-developed lessons
+to reside after the initial phases of development have been completed.
+[The Carpentries Lab][carpentries-lab] hosts a repository for open peer-review of lessons,
+and a collection of lessons that have passed through this review process 
+and can be considered ready to teach by any Instructor with sufficient knowledge of the topic.
```

Comment:
```suggestion
[The Carpentries Lab][carpentries-lab] hosts a repository for open peer-review of lessons and lessons that have passed through this review process 
and can be considered ready to teach by any Instructor with sufficient knowledge of the topic.
```
Something about this sentence structure doesn't quite make sense to me.  Are there separate locations for lessons going through peer review and lessons that have been peer reviewed?